src/PluginClient.php

```diff
@@ -62,6 +62,36 @@ public function __construct($client, array $plugins = [], array $options = [])
     }
 
     /**
+     * Adds a plugin to the client.
+     *
+     * @param Plugin $plugin
+     */
+    public function addPlugin(Plugin $plugin)
```

Comment:
I come from the symfony world and they use always `add` or `addXXX` but __appendPlugin__ sounds reasonable. 
@sagikazarmark What do you prefer?